.gitignore

```diff
@@ -1,4 +1,7 @@
 node_modules
```

Comment:
This file is, obviously, less critical...feel free to ignore the changes to ignore :-)